db/migrate/20190829112500_add_git_columns_for_coins.rb

```diff
@@ -0,0 +1,6 @@
+class AddGitColumnsForCoins < ActiveRecord::Migration[4.2]
+  def change
+    add_column :coins, :git_repo, :string
+    add_column :coins, :git_repo_type, :string
```

Comment:
Pretty sure you guys have this already but it's worth a separate migration to remove the `github_repo` column in the future.